python/pyspark/ml/feature.py

```diff
@@ -1130,8 +1143,39 @@ def numDocs(self):
         return self._call_java("numDocs")
 
 
+class ImputerParams(JavaParams, HasInputCols, HasOutputCols):
```

Comment:
Here by extending `HasOutputCols`, we do not need to add var `outputCols` by hand

Reply:
yes, I didn't have ```outputCols``` as member variable